features/locality_load_balancing.md

```diff
@@ -0,0 +1,177 @@
+[//]: # (The syntax preceeding this line is a comment marker used to help guide the author in populating this document)
+[//]: # (to github. Unlike HTML comments commonly used throughout istio.io documentation, this comment will not be rendered)
+[//]: # (by github. Comments must be separated by carriage return preceding and concluding the text and be a single line.)
+
+[//]: # (This is a living document representing the maturity of a feature. Completion of this template enables Istio work groups)
+[//]: # (to collect information on potential new functionality. This template should be completed before users are exposed to)
+[//]: # (any new experimental feature. Please complete this template during development.)
+
+[//]: # (The feature implementation section must be completed before submission of the document.)
+
+# Feature: Locality Load Balancing [Beta]
```

Comment:
Can probably simplify this template a bit by removing line 11 or line 13 like you did. 

I'd remove the word beta since this tracks the overall status of the feature, not a specific level. 

Should add this to [features.yaml](https://github.com/istio/enhancements/blob/master/features.yaml) as well.

Reply:
Will remove the word alpha/beta from the feature name. Will add to the features.yaml as well.